wp-content/themes/k1-theme/templates/page-employee-retention-credit.php

```diff
@@ -0,0 +1,74 @@
+<?php 
+
+/** 
+ * Page: ERC
+ */
+
+$content = new Content_Sections();
+k1_enqueue_page_style('employeeRetentionCredit');
+?>
+<section class="calculator">
+    <h3 class="color-erc-purple text-center">Estimate Your Employee Retention Credit Today!</h3>
+    <p class="text-center">We’re so glad you’re interested in learning more about the Employee Retention Credit available <br> to you through the Federal Government. Please complete this brief form so our ERC Team can <br> connect with you for your next step in the ERC process.</p>
+    <!-- ERC Calculator -->
+</section>
+
+<section class="erc_comparisons">
```

Comment:
Specific style thing, but use kebab-case with hyphens for base CSS classes. It helps keep the BEM system standardized (with kebab-case useable in every B,E,M section), and is generally a best-practice when writing CSS class names